src/helper.ts

```diff
@@ -7,6 +7,9 @@ export const arrayContainsEntries = <T>(collection: T[]) =>
 export const safeArrayLength = <T>(collection: T[]) =>
   collection ? collection.length : 0;
 
+export const arrayContainsElements = <T>(collection: T[]) =>
```

Comment:
This method is same as `arrayContainsEntries` isn't it?

Reply:
OOps, you are right, fixed, committing and pushing